server/utils/nunjucksSetup.ts

```diff
@@ -24,4 +24,11 @@ export default function nunjucksSetup(app: express.Application, path: pathModule
   Object.keys(mojFilters).forEach(filterName => {
     njkEnv.addFilter(filterName, mojFilters[filterName])
   })
+
+  // eslint-disable-next-line func-names
+  njkEnv.addFilter('setAttribute', function (dictionary, key, value) {
```

Comment:
why we need add this filter

Reply:
This was to solve an issue of having multiple text boxes share the same data field. It allowed us to update them with unique id's and set the govuk text box for each radio within the njk file itself, whilst also keeping the radio list separated into different sections, but treated as one list.